pkg/renovate/bump/bump.go

```diff
@@ -83,6 +84,11 @@ func New(opts ...Option) renovate.Renovator {
 		if err != nil {
 			return err
 		}
+
+		resetEpoch := true
```

Comment:
Nit: Rather than introduce `resetEpoch`, I'd hoist the `epochNode` stuff up to here and inline the conditional as `versionNode.Value != bcfg.TargetVersion`, which would bring the `if the version is changing...` comment closer to the logic/context.

Reply:
Same, lets just host the `epochNode` up.

Reply:
sgtm